graylog2-server/src/main/java/org/graylog2/indexer/EventsIndexMapping.java

```diff
@@ -0,0 +1,223 @@
+                                                                .put("index", true)
+                                                                .build())
+                                                        .build())
+                                                .build())
+                                        .add(map()
+                                                .put("fields-typed-boolean", map()
+                                                        .put("path_match", "fields_typed.boolean.*")
+                                                        .put("mapping", map()
+                                                                .put("type", "boolean")
+                                                                .put("doc_values", true)
+                                                                .put("index", true)
+                                                                .build())
+                                                        .build())
+                                                .build())
+                                        .add(map()
+                                                .put("fields-typed-ip", map()
+                                                        .put("path_match", "fields_typed.ip.*")
+                                                        .put("mapping", map()
+                                                                .put("type", "ip")
+                                                                .put("doc_values", true)
+                                                                .put("index", true)
+                                                                .build())
+                                                        .build())
+                                                .build())
+                                         */
+                                        .build())
+                                .put("properties", map()
+                                        .put("id", map()
+                                                .put("type", "keyword")
+                                                .build())
+                                        .put("creator_type", map()
+                                                .put("type", "keyword")
+                                                .build())
+                                        .put("creator_id", map()
+                                                .put("type", "keyword")
+                                                .build())
+                                        .put("timestamp", map()
+                                                .put("type", "date")
+                                                // Use the same format we use for the "message" mapping to make sure we
+                                                // can use the search.
+                                                .put("format", "yyyy-MM-dd HH:mm:ss.SSS")
```

Comment:
Should we reuse `Tools.ES_DATE_FORMAT` here?

Reply:
Totally, yes. :+1: